lib/xcpretty/parser/chunks.rb

```diff
@@ -140,6 +140,16 @@ module XCPretty
   c.line SHELL_BUILTIN
 end
 
+chunk "Process entitlements" do |c|
+  c.line /^ProcessProductPackaging "" (#{PATH})(?:-Simulated)\.xcent$/ do |f, m|
```

Comment:
I think you can `^ProcessProductPackaging .* (#{PATH}... $` in case there's something in the quotes

Reply:
yea, i was looking through our logs to try find something else and couldn't, will do that anyways.